-------------------------------------------------------------------------------
Test set: Testnow5
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.001 s <<< FAILURE! - in Testnow5
Testnow5.test_JUnit  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <This is the updated test case> but was: <This is the testcase in this class>
	at Testnow5.test_JUnit(Testnow5.java:10)

